src/lib/crypto/kimchi_bindings/stubs/kimchi_bindings.ml

```diff
@@ -178,7 +178,7 @@ module Protocol = struct
     module Fp = struct
       type nonrec t
 
-      external create : Gates.Vector.Fp.t -> int -> SRS.Fp.t -> t
+      external create : Gates.Vector.Fp.t -> int -> int -> SRS.Fp.t -> t
```

Comment:
makes me wonder if ocaml-gen should just add labels to everything... perhaps with an option or as default